dist/index.html

```diff
@@ -0,0 +1,146 @@
+        <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" class="form">
+
+            <fieldset class="group">
+                <legend>
+                  Personal information
+                </legend>
+                <div class="simple">
+                    <label for="sname">
+                      Surname:
+                        <input type="text" name="sname" id="sname" required="" autocomplete="off">
+                    </label>
+                </div>
+                <div class="line"></div>
+                <div class="simple">
+                    <label for="name">
+                      Name:
+                        <input type="text" name="name" required="" autocomplete="off" id="name">
+                    </label>
+                </div>
+                <div class="simple">
+                    <label for="old">
+                      How old are You?
+                        <input type="number" name="old" min="1" max="100" value="12" required="" id="old">
+                    </label>
+                </div>
+                <div class="simple">
+                    <label for="birth">
+                      Full date of birth:
+                        <input type="date" name="birth" required="" id="birth">
+                    </label>
+                </div>
+                <div class="line"></div>
+                <div>
+                    <label for="accept">
+                      I accept the term of the agreement:
+                        <input type="checkbox" name="accept" required="" id="accept">
+                    </label>
+                </div>
+            </fieldset>
+            <fieldset class="group">
+                <legend>
+                  Registration
+                </legend>
+                <div class="simple">
+                    <label for="email">
+                      E-mail:
+                        <input type="email" name="email" placeholder="email@example.com" required="" id="email">
+                    </label>
+                </div>
+                <div class="line"></div>
+                <div>
+                    <label for="pass">
+                      Password:
+                        <input type="password" name="pass" required="" id="pass">
+                    </label>
+                </div>
+            </fieldset>
+            <fieldset class="group">
+                <legend>
+                  An interesting fact about you!
+                </legend>
+                <div class="simple">
+                    <label for="yes">
+                      Do you love cats?
+                        <input type="radio" id="yes" name="cat" value="Yes">
+                        <label for="yes">Yes</label>
+                        <input type="radio" id="no" name="cat" value="No">
+                        <label for="no">No</label>
+                    </label>
```

Comment:
Fixes
Only inputs should be wrapped with labels, not a block of inputs
```suggestion
                    <div>
                      Do you love cats?
                        <input type="radio" id="yes" name="cat" value="Yes">
                        <label for="yes">Yes</label>
                        <input type="radio" id="no" name="cat" value="No">
                        <label for="no">No</label>
                    </div>
```